Update quoting.md with Snowflake example (#870)
dbt-labs/docs.getdbt.com: website/docs/reference/project-configs/quoting.md

### Snowflake
Set all quoting configs to `False`. This means that you cannot use reserved words as identifiers, however it's usually a good idea to avoid these reserved words anyway.

If a Snowflake source table uses a quoted database, schema, or table identifier, you can configure it in the source.yml file. [Refer to configuring quoting for more info](/reference/resource-properties/quoting).

> ```

Reviewer: ```suggestion
```

### Snowflake
Set all quoting configs to `False`. This means that you cannot use reserved words as identifiers, however it's usually a good idea to avoid these reserved words anyway.

> If you cannot change the source table name, quote the specific table name in the source.yml file:

Reviewer: ```suggestion
If a Snowflake source table uses a quoted database, schema, or table identifier, you can configure it in the source.yml file. [Refer to configuring quoting for more info](/reference/resource-properties/quoting).
```

> ```yml
> sources:
>   - name: <your_schema>
>     tables:
>       - name: group
>         identifier: GROUP
>         quoting:
>           identifier: true

Reviewer: ```suggestion
```
